qemu/tests/savevm_loadvm.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Save VM while it's running, and then load it again.
+
+    1) Launch a VM.
+    2) Save VM via human monitor while VM is running. (unsafe)
+    3) Check if it exists in snapshots.
+    4) Load VM via human monitor.
+    5) Verify kernel and dmesg.
+    6) Delete snapshot after testing.
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    snapshot_tag = "vm_" + utils_misc.generate_random_string(8)
+    os_type = params["os_type"]
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    vm.wait_for_login().close()
+    try:
+        error_context.base_context("Saving VM to %s" % snapshot_tag,
+                                   logging.info)
+        vm.monitor.human_monitor_cmd("savevm %s" % snapshot_tag)
```

Comment:
It's better to check whether "snapshot_tag"  exists in vm firstly, if it exists, maybe need to delete it.

Reply:
@wangxianxian We use a random ID in this test case, and `savevm` will replace the same tag if it exists, I think it can be left as it is. Thanks for your advice.
> savevm tag
> Create a snapshot of the whole virtual machine. If tag is provided, it is used as human readable identifier. If there is already a snapshot with the same tag, it is replaced. More info at vm_snapshots.

Reply:
Yes,  I just consider the situation that there is a snapshot that has a same ID with this one already exist in vm, but indeed the possibility is particularly small.